SciSharp.Models.Transformer/IMDbDataset.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Tensorflow;
+using Tensorflow.NumPy;
+using static Tensorflow.Binding;
+using static Tensorflow.KerasApi;
+
+namespace SciSharp.Models.Transformer
+{
+    public class IMDbDataset
+    {
+        TransformerClassificationConfig cfg;
+
+        public IMDbDataset()
+        {
+            cfg = new TransformerClassificationConfig();
+        }
+
+        public IMDbDataset(TransformerClassificationConfig cfg)
+        {
+            this.cfg = cfg;
+        }
+
+        public IMDbDataset(int vocab_size, int maxlen)
+        {
+            cfg = new TransformerClassificationConfig();
+            cfg.DatasetCfg.vocab_size = vocab_size;
+            cfg.DatasetCfg.maxlen = maxlen;
+        }
+
+        public Tensor[] GetData()
+        {
+            var dataset = keras.datasets.imdb.load_data(maxlen: cfg.DatasetCfg.maxlen);

```

Comment:
It throws error when I first run.

![image](https://github.com/SciSharp/SciSharp.Models/assets/1705364/c9dc1b21-6adb-46d2-885f-01a264614a1f)